ConsoleApp/ReportService.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using TemplateEngine.Docx;
+
+namespace ConsoleApp
+{
+    public sealed class ReportService
+    {
+        public void GenerateReport(ObjectInfo companyInfo, string output = "")
+        {
+            if (companyInfo is null)
+            {
+                return;
+            }
```

Comment:
```suggestion
        public void GenerateReport(ObjectInfo companyInfo, string output = null)
        {
```

отдельная проверка на `null` избыточна, т.к. у вас дальше выполняется повторно при вызове метода `string.IsNullOrWhiteSpace`